src/redux/books/book.js

```diff
@@ -0,0 +1,15 @@
+import { createSlice, current } from '@reduxjs/toolkit';
+
+const bookSlice = createSlice(
```

Comment:
Good job using createSlice from the redux toolkit 👍🏼

Reply:
Thank you